eclipse-scout-core/src/form/fields/groupbox/GroupBox.ts

```diff
@@ -865,3 +865,5 @@ export class GroupBox extends CompositeField implements GroupBoxModel {
 
 export type GroupBoxBorderDecoration = EnumObject<typeof GroupBox.BorderDecoration>;
 export type GroupBoxMenuBarPosition = EnumObject<typeof GroupBox.MenuBarPosition>;
+
+ObjectUuidProvider.uuidPathSkipWidgets.add(GroupBox);
```

Comment:
Could we specify this on CompositeField instead of individually on GroupBox, TabBox etc?

Reply:
No, see jsdoc of ObjectUuidProvider.uuidPathSkipWidgets